messages/cardano.proto

```diff
@@ -91,10 +98,16 @@ message CardanoSignTransactionRequest {
       repeated uint32 keypath = 1;
       bytes pool_keyhash = 2;
     }
+    message VoteDelegation {
```

Comment:
the comment above referencing cddl may be worth updating to the "conway" one where these delegation certificates are included: https://github.com/IntersectMBO/cardano-ledger/blob/cardano-ledger-conway-1.12.0.0/eras/conway/impl/cddl-files/conway.cddl